data.sql

```diff
@@ -12,4 +12,27 @@ INSERT INTO animals(name, date_of_birth, weight_kg, neutered, escape_attempts) V
 INSERT INTO animals(name, date_of_birth, weight_kg, neutered, escape_attempts) VALUES ('Squirtle', '1993-04-02', '-12.13', false, 3);
 INSERT INTO animals(name, date_of_birth, weight_kg, neutered, escape_attempts) VALUES ('Angemon', '2005-06-12', '-45', true, 1);
 INSERT INTO animals(name, date_of_birth, weight_kg, neutered, escape_attempts) VALUES ('Boarmon', '2005-06-07', '20.4', true, 7);
-INSERT INTO animals(name, date_of_birth, weight_kg, neutered, escape_attempts) VALUES ('Blossom', '1998-10-13', '17', true, 3);
\ No newline at end of file
+INSERT INTO animals(name, date_of_birth, weight_kg, neutered, escape_attempts) VALUES ('Blossom', '1998-10-13', '17', true
+
+/* New insertions (Milestone 3) */
+
+/* Insert values into orders table */
```

Comment:
- [ ] I know leaving comments in code often seems nice in describing our code, but it is often considered a good practice to avoid leaving commented code in our codebase. It would be nice you remove all comments before we merge to main what do you think? 🤔